README.md

```diff
@@ -2,12 +2,12 @@
   <br>
   <img src="https://storage.googleapis.com/golden-wind/github/omni/omni.png" alt="Omni Logo" width="100">
   <br>
-  Omni for <a href="https://www.mozilla.org/firefox/">Firefox</a>
+  Omni for <a href="https://www.mozilla.org/firefox/">Firefox</a> and <a href="https://www.thunderbird.net/pt-BR/">Thunderbird</a>
   <br>
 </h1>
 
 <p align="center">
-  <strong>Dark theme for <a href="https://www.mozilla.org/firefox/">Firefox</a></strong>
+  <strong>Dark theme for <a href="https://www.mozilla.org/firefox/">Firefox</a> and <a href="https://www.thunderbird.net/pt-BR/">Thunderbird</a></strong>
```

Comment:
I can't comment on the `team` section, but add yourself as a maintainer.
Don't forget to add your GitHub username on the `CODEOWNERS` file.

Reply:
Added in 96566293b657b52b1ff0e1eb7dd1e94ebf697082